package.json

```diff
@@ -16,7 +16,8 @@
     "onCommand:onevscode.configuration-settings",
     "onCommand:onevscode.toggle-codelens",
     "onCommand:onevscode.circle-tracer",
-    "onStartupFinished"
+    "onStartupFinished",
+    "*"
```

Comment:
I don't have much knowledge about this, but using `*` seems to fire all events...
Could you please provide some document URL for this?

Reply:
https://code.visualstudio.com/api/references/activation-events#Start-up

Reply:
I have a question. 
We were going to use `*` at first. 
However, I found the following sentence in the `onStartupFinished` description.
> This is similar to the * activation event, but it will not slow down VS Code startup.

So we added `onStartupFinished` instead of `*`.

Currently, `onStartupFinished` is added, but doesn't it work without adding `*`?

Reply:
It works with onStartupFinished too. But I think using * makes statusBar item slightly faster than using onStartupFinished...?

Reply:
We added the event to call the activate function once, and I think there is only a difference in the running time. It doesn't matter if there are two events, but I think either of them is unnecessary because the activation function runs only once anyway.
What do you think, @seanshpark ?
If `onStartupFinished` is unnecessary, I will request a new PR to remove it after this PR.

Reply:
If `onStartupFinished` is working, let's use this one. `onStartupFinished` seems not hurt performance.

Reply:
fixed it. Thank you!